torchci/scripts/deleteOldQueryLambdaVersions.mjs

```diff
@@ -0,0 +1,75 @@
+// Script to delete old versions of query lambdas
+
+import rockset from "@rockset/client";
+import { promises as fs } from "fs";
+
+const oneMonthAgo = new Date(Date.now() - 30 * 24 * 60 * 60 * 1000);
+
+async function readJSON(path) {
+  const rawData = await fs.readFile(path);
+  return JSON.parse(rawData);
+}
+
+const client = rockset.default(process.env.ROCKSET_API_KEY);
+
+const prodVersions = await readJSON("./rockset/prodVersions.json");
+Object.keys(prodVersions).forEach((workspace) => {
+  Object.entries(prodVersions[workspace]).forEach(
+    async ([queryName, jsonVersion]) => {
+      try {
+        const versions = await client.queryLambdas.listQueryLambdaVersions(
+          workspace,
+          queryName
+        );
+        const prodVersion = (
+          await client.queryLambdas.getQueryLambdaTagVersion(
+            workspace,
+            queryName,
+            "prod"
+          )
+        ).data.version.version;
```

Comment:
What happens if there's no label with a "prod" tag?

Reply:
All lambdas in prodversions.json should have a prod tag somewhere, added by a different script.  

the try catch should ensure that other lambdas can continue being deleted.  I'm willing to eat the additional cost of manually deleting versions if something fails